labelbox/schema/project.py

```diff
@@ -177,6 +177,51 @@ def export_labels(self, timeout_seconds=60):
                          self.uid)
             time.sleep(sleep_time)
 
+    def export_issues(self, status=None, timeout_seconds=60):
+        """ Calls the server-side Issues exporting that generates a JSON
+        payload, and returns the URL to that payload.
+
+        Args:
+            timeout_seconds (float): Max waiting time, in seconds.
+            status (string): valid values: Open, Resolved
+        Returns:
+            URL of the data file with this Project's issues. If the server didn't
+            generate during the `timeout_seconds` period, None is returned.
+        """
+        sleep_time = 2
+        id_param = "projectId"
+        status_param = "status"
+        query_str = """query getProjectIssues($%s: ID!, $%s: IssueStatus) {
+            project(where: { id: $%s }) {
+                issueExportUrl(where: { status: $%s })
+            }
+        }""" % (id_param, status_param, id_param, status_param)
+
+        valid_statuses = [None, "Open", "Resolved"]
+
+        if status not in valid_statuses:
+            raise ValueError("status must be in {}. Found {}".format(
+                valid_statuses, status))
+
+        while True:
+            res = self.client.execute(query_str, {
+                id_param: self.uid,
+                status_param: status
+            })
+            print(res)
+            res = res.get('project').get('issueExportUrl')
```

Comment:
I think there is potential for this to quietly break in the future. If the issueExportKey is missing for any reason this will silently fail. Is there a more specific way to see if we should retry? Also why not use res['project'] instead of res['project'] so that it throws a KeyError instead of an AttributeError?

Reply:
I would retry if res.get('issueExportUrl') return None (key is missing) or say '' string is returned for URL

Reply:
I am just wondering if this can be specific about when to retry. Wouldn't there only be one condition to retry on? An empty string or a missing key? It doesn't make sense for both of those conditions. I am worried that checking for a missing key to retry could cause problems. If the key always exists and the retry depends on whether the string is empty or not, then you should change this to explicitly check for an empty string.

Reply:
E.g. the[project.export_labels](https://github.com/Labelbox/labelbox-python/blob/0cea76258db6fedbef54d4b69518b5e320121fc5/labelbox/schema/project.py#L169)  checks for the server to respond with whether or not it should poll. Is this not available for issues? I assume the implementation of the exporter is similar.